### DISCUSSION TITLE
Whoogle and Tor

### DISCUSSION THREAD

**Discussion by arconan42** [benbusby/whoogle-search#618] [Q&A] [answered]
Hey, just set up whoogle in docker on an old optiplex and am impressed so far, so thankyou for this project.

However I cant seem to set up whoogle to use tor network. I have even set the environmental during install and have tor available on the host but when I use whoogle it says tor is unavailable.

I am sure its just myself doing something wrong in the steps but I cant pin down what I am doing wrong.

I install tor on the host I setup docker and whoogle on, and install whoogle via below manual docker install:

docker run --publish 5000:5000 --detach --name whoogle-search \
  -e WHOOGLE_CONFIG_COUNTRY=australia \
  -e WHOOGLE_CONFIG_LANGUAGE=english \
  -e WHOOGLE_CONFIG_SEARCH_LANGUAGE=english \
  -e WHOOGLE_CONFIG_THEME=dark \
  -e WHOOGLE_CONFIG_TOR=yes \
  whoogle-search:1.0

Any thoughts?

**Comment by Albonycal** [comment]
any errors?

**Reply by arconan42** [reply]
no errors, just doesnt seem to want to activate tor searching

**Comment by gauravdash** [comment, marked as answer]
Hi,

Whoogle docker image includes tor binary and is started when docker container is run. 
There is no need to run tor separately on the host.
See the Dockerfile for reference.

Thanks.
